operators/tokenizer/bpe_kernels.cc

```diff
@@ -276,7 +276,14 @@ std::vector<int64_t> KernelBpeTokenizer::Tokenize(ustring& input,
     }
 
     while (static_cast<int64_t>(res.size()) < max_length) {
-      auto [b, tok] = regcmp.GetNextToken();
+      std::string regex_expr = "";
+      if (ModelName() == kModel_Llama){
+        regex_expr = "(?i:'s|'t|'re|'ve|'m|'ll|'d)|[^\\r\\n\\p{L}\\p{N}]?\\p{L}+|\\p{N}{1,3}| ?[^\\s\\p{L}\\p{N}]+[\\r\\n]*|\\s*[\\r\\n]+|\\s+(?!\\S)|\\s+";
```

Comment:
I saw this pattern appeared several time, can we define a const string variable for it?

Reply:
good point, done!